src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -214,23 +216,47 @@ class JesBackend extends Backend with LazyLogging {
     }
   }
 
-  private def uploadCommandScript(backendCall: BackendCall, command: String): Unit = {
+  private def uploadCommandScript(backendCall: BackendCall, command: String): Try[Unit] = {
     val fileContent =
       s"""
          |#!/bin/bash
          |$command
          |echo $$? > ${JesBackendCall.RcFilename}
        """.stripMargin.trim
-    JesConnection.storage.uploadObject(backendCall.gcsExecPath, fileContent)
+
+    def attemptToUploadObject(priorAttempt: Option[Unit]) = JesConnection.storage.uploadObject(backendCall.gcsExecPath, fileContent)
+
+    TryUtil.retryBlock(
+      fn = attemptToUploadObject,
+      retries = 10,
+      pollingInterval = 5 seconds,
+      pollingBackOffFactor = 1,
```

Comment:
No reason, I figured the backoff isn't really adding value here... do you have suggestions for the values of these numbers?